### ISSUE TITLE
fixing class diagram 

### ISSUE THREAD

**Issue by DanShai** [knsv/mermaid#794]
Hi ,
1- after debugging the class diagram there are many issues in the code mainly in the path.ponits assigned to and other variables  like   `lineData = path.points; `   return   NaN in the x and y of the points  to the svg  path  hence the messed up diagrams.

 
2- if your using react like me  and getting this kind of errors  Expecting 'CLASS_DIAGRAM', got 'NEWLINE'  you should write the graph like this :
for example :
```
 const graph = `classDiagram
  Class01 <|-- AveryLongClass : Cool
  Class03 *-- Class04
  Class01 : int chimp
  Class01 : int gorilla`;
```
no space nor new line at the start or the end
hope this helps! I saw many have this issue.

PS:  for react users I created a Mermaid hook for easy to use in your app, a screenshot for the fix for class diagram can be found in the link below   : 
[https://github.com/DanShai/useMermaid-hook](https://github.com/DanShai/useMermaid-hook)

**Comment by DanShai** [comment]
I fixed it and working cool for me :)  I will make a PR soon

**Comment by knsv** [comment]
Thanks again @DanShai. I am closing this one as the PR is in place and released in 8.1.0.
